### PR TITLE
Migrate a set of glued contact tests in combined to use TM

### PR DESCRIPTION
**Pull request by recuero**
ref #6978

### CODE REVIEW

**Comment by bwspenc** [comment]
@permcody Could you please add @recuero to the INL developers group?  He's a new employee in our department.

File: modules/combined/test/tests/glued_contact/sm/glued_contact_mechanical_constraint_test_sm.i [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    youngs_modulus = 1e6
+    poissons_ratio = 0.3
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  #petsc_options_iname = '-pc_type -pc_hypre_type -snes_type -snes_ls -snes_linesearch_type -ksp_gmres_restart'
```
Comment (bwspenc) [inline modules/combined/test/tests/glued_contact/sm/glued_contact_mechanical_constraint_test_sm.i]: Remove commented out lines

File: modules/combined/test/tests/glued_contact/sm/glued_contact_mechanical_constraint_test_sm.i [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    slave = 3
+    penalty = 1e6
+    model = glued
+    formulation = kinematic
+    system = constraint
+  [../]
+[]
+
+[BCs]
+
```
Comment (bwspenc) [inline modules/combined/test/tests/glued_contact/sm/glued_contact_mechanical_constraint_test_sm.i]: remove this blank line

File: modules/combined/test/tests/glued_contact/sm/glued_contact_test_sm.i [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    youngs_modulus = 1e6
+    poissons_ratio = 0.3
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+
+  #petsc_options_iname = '-pc_type -pc_hypre_type -snes_type -snes_ls -snes_linesearch_type -ksp_gmres_restart'
```
Comment (bwspenc) [inline modules/combined/test/tests/glued_contact/sm/glued_contact_test_sm.i]: Remove commented out lines

**Review by bwspenc — CHANGES_REQUESTED** [review]
Just a few changes.  Looks good!

**Comment by recuero** [comment]
Suggested format changes included in amended commit.

**Review by bwspenc — APPROVED** [review]

**Pull request merged**